gen-compdocs/comps/gen_kube_docs.go

```diff
@@ -45,27 +45,23 @@ func GenerateFiles(path, module string) {
 	case "kube-apiserver":
 		pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
 		apiserver := apiservapp.NewAPIServerCommand()
-		GenMarkdownTree(apiserver, outDir, true)
+		generators.GenMarkdownTree(apiserver, outDir, true)
 
 	case "kube-controller-manager":
 		pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
 		controllermanager := cmapp.NewControllerManagerCommand()
-		GenMarkdownTree(controllermanager, outDir, true)
+		generators.GenMarkdownTree(controllermanager, outDir, true)
 
 	case "kube-proxy":
-		proxy := proxyapp.NewProxyCommand()
```

Comment:
What is supposed to happen in the `kube-proxy` case?

Reply:
good catch.